tests/components/group/test_init.py

```diff
@@ -1884,3 +1968,210 @@ async def test_unhide_members_on_remove(
     # Check the group members are unhidden
     assert entity_registry.async_get(f"{group_type}.one").hidden_by == hidden_by
     assert entity_registry.async_get(f"{group_type}.three").hidden_by == hidden_by
+
```

Comment:
Do we need to test a group of group entities here?

Reply:
We have these test for testing groups in groups:
https://github.com/home-assistant/core/blob/b3c1a86194c83da647f8e1acf11b296ccfac08da/tests/components/group/test_init.py#L1587-L1668

Do we need more tests?